index.js

```diff
@@ -1,18 +1,25 @@
-const readline = require('readline');
+const readline = require("readline");
 const rl = readline.createInterface(process.stdin, process.stdout);
+start();
 
 function ask(questionText) {
   return new Promise((resolve, reject) => {
     rl.question(questionText, resolve);
   });
 }
 
-start();
-
 async function start() {
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
-  console.log('You entered: ' + secretNumber);
-  // Now try and complete the program.
+  let done = false;
+  while (!done) {
```

Comment:
Flags using negation are more confusing that flags using the positive case. Consider changing.

Such as:

```js
while( keepGoing) { /* doStuff() */ }
```